src/components/report/Summary.js

```diff
@@ -9,7 +9,32 @@ export default {
 	},
 	computed: {
 		featureLength() {
-			return Object.keys(this.features).length;
+			const count = Object.keys(this.features).length;
+			return count || "no";
+		},
+		axesCount() {
```

Comment:
This could maybe be simplified as:

```js
if (this.font.variable) {
  return this.font.variable.axes.length;
} else {
  return "no";
}
```

I don't think it's possible for a variable font to have 0 axes, right?

Reply:
That's true, then even the `else` clause can go.